lib/fluent/gelf_util.rb

```diff
@@ -67,7 +67,7 @@ def make_gelfentry(tag,time,record, conf = {})
         end
       end
 
-      if !gelfentry.key?('short_message') or gelfentry['short_message'].to_s.empty? then
+      if !gelfentry.key?('short_message') or gelfentry['short_message'].to_s.empty? or gelfentry['short_message'].to_s.strip.empty? then
```

Comment:
I think it would be enought o just do `gelfentry['short_message'].to_s.strip.empty?`, I don't see a case where it should cause an error.

Reply:
Good catch, yes!